src/mbgl/programs/symbol_program.cpp

```diff
@@ -1,14 +1,21 @@
+#include <iostream>
 #include <mbgl/programs/symbol_program.hpp>
 #include <mbgl/renderer/render_tile.hpp>
 #include <mbgl/map/transform_state.hpp>
 #include <mbgl/style/layers/symbol_layer_impl.hpp>
+#include <mbgl/util/enum.hpp>
 
 namespace mbgl {
 
 using namespace style;
 
 static_assert(sizeof(SymbolLayoutVertex) == 16, "expected SymbolLayoutVertex size");
 
+MBGL_DEFINE_ENUM(SymbolSDFPart, {
```

Comment:
`MBGL_DEFINE_ENUM` is needed only where enum ↔ string conversion is required, which isn't the case with `SymbolSDFPart`.